src/core/mantle/types/appearance.ts

```diff
@@ -84,7 +84,9 @@ export type EllipsoidAppearance = {
 };
 
 export type ModelAppearance = {
+  // For compat
```

Comment:
Is the whole ModelAppearance type "for compatiblility"? Or just the model prop? If just the model prop, move this to that line (like the `// default: true` below on line 97) so that it is clearer.

Reply:
https://github.com/reearth/reearth-web/pull/387/commits/9cfe5d7c35485681e0a37a690859e049bf3425ec
I'm sorry I ended up merging this PR unintentionally, so I fixed this comment in another PR...